admin-ui/plugins/auth-server/helper/validations.js

```diff
@@ -0,0 +1,25 @@
+// LDAP validation schema using Yup
+import * as Yup from 'yup'
+import { STRINGS } from './constants'
+
+export function getLdapValidationSchema() {
+  const e = STRINGS.authn.ldap.errors
+  return Yup.object().shape({
+    configId: Yup.string().min(1, e.acr).required(e.acr),
+    bindDN: Yup.string().min(1, e.bind_dn).required(e.bind_dn),
+    bindPassword: Yup.string().min(1, e.bind_password).required(e.bind_password),
+    servers: Yup.array()
+      .of(Yup.string().min(1, e.remote_ldap_server).required(e.remote_ldap_server))
+      .min(1, e.remote_ldap_server)
+      .required(e.remote_ldap_server),
```

Comment:
Consider adding server URL format validation

The servers array validation only checks for non-empty strings. Consider adding URL format validation to ensure valid LDAP server addresses.



```diff
     servers: Yup.array()
-      .of(Yup.string().min(1, e.remote_ldap_server).required(e.remote_ldap_server))
+      .of(
+        Yup.string()
+          .min(1, e.remote_ldap_server)
+          .matches(
+            /^(ldap|ldaps):\/\/[^\s]+$/,
+            'Server must be a valid LDAP URL (ldap:// or ldaps://)'
+          )
+          .required(e.remote_ldap_server)
+      )
       .min(1, e.remote_ldap_server)
```